src/sass/components/button.scss

```diff
@@ -82,17 +82,124 @@
   }
 }
 
+@mixin alt-tetra-button($button){
```

Comment:
It looks simple in SCSS. But since only the LESS files are to be modified and not the SCSS output from less-to-sass, it will be a nightmare.
I had a look on grunt-less-to-sass and, unfortunately, the replacement for the LESS 'and' keyword doesn't exist which lead to badly generated SCSS files and the CSS compilation fail... leading to the impossibility to launch the the framework.